Reflection/NPC.simba

```diff
@@ -151,3 +151,34 @@ begin
   RSTypeArray(NPCs).Free;
   SetLength(Result, J);
 end;
+
+function R_DismissRandoms(TakeLamp: Boolean = True) : Boolean;
+var
+  AllNPCs : array of RSNPC;
+  EachNPC : RSNPC;
+  NPCTPA : TPointArray;
+  EachNPCDef: RSNPCDefinition;
+  MyName: String;
+  TimeToWait: Int32;
+begin
+  AllNPCs := RSNPC.AllNPCs;
+  MyName := RSPlayer.Me.Name;
+  for EachNPC in AllNPCs do begin
+    EachNPCDef := EachNPC.Definition;
+    if (EachNPC.SpokenText <> '') and (MyName in EachNPC.SpokenText) then begin
+      Wait(1200, 5000);
+      NPCTPA := EachNPC.ToTPA;
+      Mouse.Move(MiddleTPA(NPCTPA), 5, true);
```

Comment:
I like it. Better than using just the tile location to click the NPC.

Reply:
It's still pretty rudimentary, I just lifted it from a code snippet in discord that @Brandon-T or someone posted. Works well enough for now